src/include/catalog/index_schema.h

```diff
@@ -23,18 +23,19 @@ class TpccPlanTest;
 
 namespace noisepage::storage {
 class RecoveryManager;
-}
+}  // namespace noisepage::storage
 
 namespace noisepage::tpcc {
 class Schemas;
-}
+}  // namespace noisepage::tpcc
```

Comment:
I know this only popped in the diff because of a `clang-format` run, but this falls into the same category of the TPC-H friendship on `proc`.  I'm fine with leaving a TODO and issue, but I do think we need to clearly mark this as the "wrong" approach for future benchmarks/tests.

Reply:
Happily removable without further work.